text/0000_require_minimal_ruby_version.md

```diff
@@ -0,0 +1,38 @@
+- Feature Name: mandatory_required_ruby_version
+- Start Date: 2020-04-13
+- RFC PR: 
+- Bundler Issue: 
+
+# Summary
+
+Calling `spec.required_ruby_version` in a gemspec should be mandatory.
+
+# Motivation
+
+My current problem is that many gems are not publishing their `required_ruby_version`.
+
+This means that a Gemfile/gemspec must know what is the maximum version of a gem for a given version of Ruby, instead of bundler figuring it out for us.
+
+It takes about one minute for the gem author to add the line. When the author doesn't, everyone that uses the gem, directly or indirectly, may have to figure out the maximum version and specify it in their own gemfiles.
+
+Note that there is currently no good way to correct this mistake. Once it is done, the only option [is to yank the gem](https://github.com/rubygems/rubygems/issues/1506#issuecomment-188472423) which is typically not what one wants to do.
+
+`required_ruby_version` is currently in the "Optional" section of [the spec](https://guides.rubygems.org/specification-reference/). It's not even in the "Recommended" section!
+
+It should be mandatory and enforced.
+
+# Guide-level explanation
+
+Building a gem that does not specify it would fail, same as the other required fields.
+
+# Reference-level explanation
+
+Building a gem that does not specify it would fail, same as the other required fields.
+
+# Drawbacks
+
+I can not think of a single valid reason not to specify a required ruby version.
```

Comment:
Could you please share your practical concerns so I can amend this section?

Reply:
The boilerplate circumvention is a large part, and then I had in mind how this would fit in with pre-existing gems but I lost steam because even now the argument I have in mind there would take a while to wrap my head around communicating - the idea being that it’s a rough transition for small benefit or it’s not going to be any benefit... but it’s 3am, and I was hoping the line of reasoning I already tried was enough for now...